### PR TITLE
text utilities

### PR DESCRIPTION
**Pull request by SumitSaundadkar**
text-utilities

### CODE REVIEW

**Review by TusharJade — COMMENTED** [review]
looks good to me

**Review by Ananyamadhu08 — APPROVED** [review]
looks good, can go ahead and merge 👍

File: components/text-utilities.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  }
+  .text-gray {
+    color: var(--btn-secondary-color);
+  }
+  .line-through {
+    text-decoration: line-through;
+  }
+  .text-center {
+    text-align: center;
+  }
```
Comment (deekshithmd) [inline components/text-utilities.css:80]: Code looks good to merge

**Review by deekshithmd — APPROVED** [review]

File: components/text-utilities.css (hunk 1 above)
Comment (SumitSaundadkar) [reply components/text-utilities.css:80]: thank you.

**Pull request merged**